src/crypto/tls/handshake_server_tls13.go

```diff
@@ -892,6 +942,11 @@ func (hs *serverHandshakeStateTLS13) sendServerParameters() error {
 }
 
 func (hs *serverHandshakeStateTLS13) requestClientCert() bool {
+	for _, flag := range hs.tlsFlags {
+		if flag == ExperimentalFlagSupportMTLS {
+			return true
+		}
+	}
```

Comment:
Suggestion, feel free to ignore: I feel like it ought to be possible to do this without iterating over the set flags. Can't we just do some bit fiddling on the the encoded TLS flags to see if the req mTLS flag is set?